From efd713654540ed680414d739a684168666eaf6b6 Mon Sep 17 00:00:00 2001
From: Kjartan Maraas <kmaraas@gnome.org>
Date: Wed, 6 Oct 2010 11:00:56 +0200
Subject: [PATCH 02/14] Remove obsolete ifdef

---
 include/orbit/util/basic_types.h | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/include/orbit/util/basic_types.h b/include/orbit/util/basic_types.h
index 8702276..c8562ca 100644
--- a/include/orbit/util/basic_types.h
+++ b/include/orbit/util/basic_types.h
@@ -34,7 +34,6 @@ typedef CORBA_wchar *CORBA_wstring;
 typedef struct CORBA_Object_type *CORBA_Object;
 #endif
 
-#ifdef G_HAVE_GINT64
 #define HAVE_CORBA_LONG_LONG
 
 /*
@@ -44,13 +43,6 @@ typedef struct CORBA_Object_type *CORBA_Object;
 typedef gint64  CORBA_long_long;
 typedef guint64 CORBA_unsigned_long_long;
 
-#else
-#error ""
-#error "You don't G_HAVE_GINT64 defined in glib."
-#error "Please make sure you don't have an old glibconfig.h lying around."
-#error ""
-#endif
-
 G_END_DECLS
 
 #endif
-- 
1.8.3

