diff -ru gnokii-0.6.29.dfsg.before/common/phones/pcsc.c gnokii-0.6.29.dfsg/common/phones/pcsc.c
--- gnokii-0.6.29.dfsg.before/common/phones/pcsc.c	2010-02-23 00:05:36.000000000 +0100
+++ gnokii-0.6.29.dfsg/common/phones/pcsc.c	2010-08-18 22:34:39.000000000 +0200
@@ -199,7 +199,6 @@
 	case SCARD_W_UNPOWERED_CARD:
 	case SCARD_W_RESET_CARD:
 	case SCARD_W_REMOVED_CARD:
-	case SCARD_W_INSERTED_CARD:
 	 	return GN_ERR_SIMPROBLEM;
 	case SCARD_E_NO_MEMORY:
 		return GN_ERR_MEMORYFULL;
