diff -Naur pcmanfm-20110422_3f899d1.orig/src/main-win.c pcmanfm-20110422_3f899d1/src/main-win.c
--- pcmanfm-20110422_3f899d1.orig/src/main-win.c	2011-04-22 22:28:57.000000000 +0200
+++ pcmanfm-20110422_3f899d1/src/main-win.c	2011-05-01 16:47:00.000000000 +0200
@@ -568,19 +568,6 @@
     /* the location bar */
     win->location = fm_path_entry_new();
     g_signal_connect(win->location, "activate", on_location_activate, win);
-    if(geteuid() == 0) /* if we're using root, give the location entry a different color */
-    {
-        GtkStyle* style = gtk_rc_get_style_by_paths(
-                            gtk_settings_get_for_screen(gtk_widget_get_screen(win->location)),
-                            "gtk-tooltip", NULL, G_TYPE_NONE);
-        if(style)
-        {
-            gtk_widget_modify_base(win->location, GTK_STATE_NORMAL, &style->bg[GTK_STATE_NORMAL]);
-            gtk_widget_modify_fg(win->location, GTK_STATE_NORMAL, &style->fg[GTK_STATE_NORMAL]);
-            gtk_entry_set_icon_from_stock(GTK_ENTRY(win->location), GTK_ENTRY_ICON_PRIMARY, GTK_STOCK_DIALOG_WARNING);
-        }
-        gtk_entry_set_icon_tooltip_text(GTK_ENTRY(win->location), GTK_ENTRY_ICON_PRIMARY, _("You are in super user mode"));
-    }
 
     toolitem = gtk_tool_item_new();
     gtk_container_add( GTK_CONTAINER(toolitem), win->location );
